tools/cldr-apps/js/src/esm/cldrAuth.mjs

```diff
@@ -0,0 +1,46 @@
+import * as cldrClient from "../esm/cldrClient.mjs";
+import * as cldrStatus from "../esm/cldrStatus.mjs";
+
+/**
+ * Get the oauth login URL
+ * @param {object} o options bag
+ * @param {string} o.service which service, currently only 'github' is accepted
+ * @param {string} o.intent what the login URL is used for, currently only 'cla'
+ * @param {boolean} o.relogin if true, attempt re-login
+ */
+export async function getLoginUrl(o) {
+  const { service, intent, relogin } = o;
+  if (service !== "github")
+    throw Error(`only support service='github' but got ${service}`);
+  if (intent !== "cla")
+    throw Error(`only support intent='cla' but got ${intent}`);
+  const client = await cldrClient.getClient();
+  const { url } = (await client.apis.auth.oauthUrl()).body;
+  const u = new URL(url);
+  // TODO: may move this into the GithubLoginFactory#getLoginUrl()
```

Comment:
TODO comment ideally should include ticket number

Reply:
I will change this to a see also.